### DISCUSSION TITLE
Root.NSupply.Uptake is always zero

### DISCUSSION THREAD

**Discussion by BrianCollinss** [APSIMInitiative/ApsimX#8902] [Q&A] [answered]
What is the rationale here?

![image](https://github.com/APSIMInitiative/ApsimX/assets/39573622/5e037cf1-da61-4ba9-b9f2-12556d142d68)

**Comment by HamishBrownPFR** [comment, marked as answer]
N uptake supply by roots is calculated by the soil arbitrator so is set to zero here
